pkg/k8s/apis/cilium.io/register.go

```diff
@@ -6,4 +6,14 @@ package ciliumio
 const (
 	// CustomResourceDefinitionGroup is the name of the third party resource group
 	CustomResourceDefinitionGroup = "cilium.io"
+
+	// CustomResourceDefinitionSchemaVersionKey is key to label which holds the CRD schema version
+	CustomResourceDefinitionSchemaVersionKey = "io.cilium.k8s.crd.schema.version"
+
+	// CustomResourceDefinitionSchemaVersion is semver-conformant version of CRD schema
+	// Used to determine if CRD needs to be updated in cluster
+	//
+	// Maintainers: Run ./Documentation/check-crd-compat-table.sh for each release
+	// Developers: Bump patch for each change in the CRD schema.
+	CustomResourceDefinitionSchemaVersion = "1.27.0"
```

Comment:
I think we should keep the `"1.26.7"`. Similar the version set in VERSION, which is 1.13.99, not 1.14.0.

Reply:
Ack, it's also the one that also *should* be in active use.